### PR TITLE
doc: replace documentation with the handbook

### PR DESCRIPTION
**Pull request by saghul**
- Leave a link on doc/README
- Moved the cloud API swagger file to resources/
- Merged the coding conventions with CONTRIBUTING.md

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -1,47 +1,3 @@
...
-* [Video: How to Load Balance Jitsi Meet](https://jitsi.org/blog/tutorial-video-how-to-load-balance-jitsi-meet/)
-* [Video: Scaling Jitsi Meet in the Cloud](https://jitsi.org/blog/new-tutorial-video-scaling-jitsi-meet-in-the-cloud/)
-* [Configuring a video SIP gateway](https://github.com/jitsi/jitsi-meet/blob/master/doc/sipgw-config.md)
-* [Enabling speaker stats](https://github.com/jitsi/jitsi-meet/blob/master/doc/speakerstats-prosody.md)
-* [Enabling TURN](https://github.com/jitsi/jitsi-meet/blob/master/doc/turn.md)
-* [Networking FAQ](https://github.com/jitsi/jitsi-meet/blob/master/doc/faq.md)
-* [Cloud APIs](https://github.com/jitsi/jitsi-meet/blob/master/doc/cloud-api.md)
-* [Manual Installation](https://github.com/jitsi/jitsi-meet/blob/master/doc/manual-install.md)
-* [Scalable Installation](https://github.com/jitsi/jitsi-meet/blob/master/doc/scalable-installation.md)
+The Jitsi documentation has been moved to [The Handdbook](https://jitsi.github.io/handbook/).
```
Comment (l3d00m) [inline doc/README.md:3]: Handdbook has a typo ('d' twice)

File: doc/quick-install.md [hunk 2]
```diff
@@ -1,162 +1,3 @@
...
-If you run into problems, one thing to try is using a different web browser. Some versions of some browsers are known to have issues with Jitsi Meet. You can also visit https://test.webrtc.org to test your browser's [WebRTC](https://en.wikipedia.org/wiki/WebRTC) support.
-
-Another place to look is the various log files:
-
-```
-/var/log/jitsi/jvb.log
-/var/log/jitsi/jicofo.log
-/var/log/prosody/prosody.log
-```
+This document has been moved [here](https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-quickstart).
```
Comment (luixxiul) [inline doc/quick-install.md:3]: Have these proposals been cherry-picked and reflected to this new handbook?
https://github.com/jitsi/jitsi-meet/pulls?q=is%3Apr+is%3Aopen+quick-install.md

File: doc/scalable-installation.md [hunk 3]
```diff
@@ -1,166 +0,0 @@
-# Scalable Jitsi installation
```
Comment (luixxiul) [inline doc/scalable-installation.md:1]: See above: https://github.com/jitsi/jitsi-meet/pull/6278

File: doc/faq.md [hunk 4]
```diff
@@ -1,7 +0,0 @@
-**1. How to tell if my server instance is behind NAT?**
```
Comment (luixxiul) [inline doc/faq.md:1]: This one too.

This PR (https://github.com/jitsi/jitsi-meet/pull/5960) should be reflected to https://jitsi.github.io/handbook/docs/faq

File: doc/quick-install.md (hunk 2 above)
Comment (saghul) [reply doc/quick-install.md:3]: Yes the plan is to move all PRs there. If you want to help with that, awesome!

**Comment by saghul** [comment]
I’ll go through the open PRs and apply then to the handbook, if nobody beats me to it :-)

File: doc/quick-install.md (hunk 2 above)
Comment (luixxiul) [reply doc/quick-install.md:3]: Right, please let me take care of that. I'm cherry-picking them and fixing conflicts as many as possible 😄

**Pull request merged**